lib/util.js

```diff
@@ -491,16 +491,12 @@ const util = class util {
     if ( queryDup.order_by === "" ) {
       delete queryDup.order_by;
     }
-    if ( _.isEmpty( queryDup.lat ) && _.isEmpty( queryDup.lat ) ) {
```

Comment:
I believe the root cause of the new symptom is that for v2, parameters are coerced into their defined data type and not all strings coming from the request parameter string as is the case with v1. The lodash `_.isEmpty` method considers numbers to be empty, resulting in v2 requests with lat/lng having their lat/lng ignored when generating a cache key. All else looks good, so I'll merge this but might restore this bit adding in an additional check of `&& !_.isNumber`